cmd/bosun/expr/parse/parse.go

```diff
@@ -31,13 +31,14 @@ type Tree struct {
 }
 
 type Func struct {
-	Args     []models.FuncType
-	Return   models.FuncType
-	Tags     func([]Node) (Tags, error)
-	F        interface{}
-	VArgs    bool
-	VArgsPos int
-	Check    func(*Tree, *FuncNode) error
+	Args      []models.FuncType
+	Return    models.FuncType
+	Tags      func([]Node) (Tags, error)
+	F         interface{}
+	VArgs     bool
+	VArgsPos  int
+	VArgsOmit bool
```

Comment:
How about inverting it. Maybe `VArgsRequired` or even `MinVArgs` would be clearer.